DRODLib/Character.cpp

```diff
@@ -2421,6 +2424,8 @@ void CCharacter::Process(
 								}
 							}
 						}
+						else if (IsAlive())
```

Comment:
Suggest a minor refactor to make this easier to grok:

Back at new line 2402, how about:
```
if (IsAlive()) {
  if (!this->bVisible) {
     room.KillInvisibleCharacter(this);
  } else {
     ...everything else...
  }
}
```